portworx-shared-volumes/step6.md

```diff
@@ -0,0 +1 @@
+>>Q11. Mount the volume 'shared_volume' on node03 at the directory /var/lib/osd/mounts/node03<<
```

Comment:
these look like duplicate questions to lighthouse lab and replication sets labs https://github.com/portworx/katacoda-scenarios/pull/57/files#diff-2235f75c8d1cfc21c1bdff5914f3ec73 ?

Reply:
This scenario is the only one using this step -  mounting shared volumes. 

This step is not used in the index.json for lighthouse or replication set labs. 

I will clean it up, but this can be ignored.